tests/helpers/session.py

```diff
@@ -523,12 +534,12 @@ def setup_backchannel(self):
         _, self.backchannel_port = self.backchannel_socket.getsockname()
         self.backchannel_socket.listen(0)
 
-        backchannel_thread = threading.Thread(target=self._backchannel_worker, name='bchan#%d listener' % self.ptvsd_port)
+        backchannel_thread = threading.Thread(target=self._backchannel_worker, name='bchan#%d listener' % self.backchannel_port)
```

Comment:
This is actually supposed to use `ptvsd_port`, so that you can correlate backchannels and ptvsd instances using them, when looking at test log. Backchannel ports themselves are random (and hence useless).